packages/react-components/react-carousel-preview/library/src/components/CarouselAutoplayButton/CarouselAutoplayButton.types.ts

```diff
@@ -1,17 +1,31 @@
+import { ARIAButtonSlotProps } from '@fluentui/react-aria';
+import { ButtonProps, ButtonSlots, ButtonState } from '@fluentui/react-button';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
-export type CarouselAutoplayButtonSlots = {
-  root: Slot<'div'>;
+export type CarouselAutoplayButtonSlots = ButtonSlots & {
+  root: NonNullable<Slot<ARIAButtonSlotProps>>;
 };
 
 /**
  * CarouselAutoplayButton Props
  */
-export type CarouselAutoplayButtonProps = ComponentProps<CarouselAutoplayButtonSlots> & {};
+export type CarouselAutoplayButtonProps = ButtonProps &
+  ComponentProps<CarouselAutoplayButtonSlots> & {
+    /**
+     * Controls whether autoplay will initialized as true or false
+     * Default: true
```

Comment:
Are we sure we want to default it to true? I feel like the more accessible choice would be false, but you might have already been through that discussion and I'm just coming up to speed.

Reply:
I think we could use either here as the component itself is optional and also the core requirement for autoplay to be enabled, it has a start delay timer so it won't kick off right away but should be passively 'on' without requiring interaction if this button exists.